xsd-fu/python/generateDS/generateDS.html

```diff
@@ -268,7 +268,7 @@ <h1 class="title">generateDS -- Generate Data Structures from XML Schema</h1>
 </ul>
 </li>
 <li><a class="reference internal" href="#common-problems" id="id16">5&nbsp;&nbsp;&nbsp;Common problems</a><ul class="auto-toc">
-<li><a class="reference internal" href="#namespace-prefix-mis-match" id="id17">5.1&nbsp;&nbsp;&nbsp;Namespace prefix mis-match</a></li>
+<li><a class="reference internal" href="#namespace-prefix-mismatch" id="id17">5.1&nbsp;&nbsp;&nbsp;Namespace prefix mismatch</a></li>
```

Comment:
Can this be reverted please? This is not documentation and so should be ignored.